doc/changes/names.inc

```diff
@@ -389,3 +389,7 @@
 .. _Jack Zhang: https://github.com/jackz314
 
 .. _Felix Klotzsche: https://github.com/eioe
+
+.. _David Julien: https://github.com/Swy7ch
+
+.. _Romain Derollepot: https://github.com/rderollepot
```

Comment:
@Swy7ch @rderollepot let me know if you want different URLs here

Reply:
That is great for me, thanks 👍 

I also want to take this opportunity to say that I am really grateful for your help @larsoner and @rob-luke in seing that the great contribution @Swy7ch started almost a year ago can come to an end, improving it with new features along the way. We are still very new to MNE in the lab (it started with David's internship), and I am not sure I would have been able to finish it myself (or it would have take me ages 😅).